test/e2e/performance/bpmn.navigation.performance.test.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { BpmnDiagramPreparation, BpmnLoadMethod, delay, getSimplePlatformName, PageTester } from '../helpers/visu-utils';
+import { calculateMetrics, ChartData, PerformanceMetric } from '../helpers/perf-utils';
+import { Mouse } from 'puppeteer';
+import * as fs from 'fs';
+// FIXME - to be fixed when new release of puppeteer comes out
+// wheel is added in version @types/puppeteer 2.1.5 but for some reason not in 3.0.2
+// perhaps will be soon available in 3.0.3
+// @see https://github.com/puppeteer/puppeteer/pull/6141/files
+interface MouseWheelOptions {
```

Comment:
This is now available directly with puppeteer types, see 2db8be7f48dd7a5ce68f9befd926588f97999411

Reply:
yes finally :), rebased and fixed